test/e2e/testnet/testnet.go

```diff
@@ -141,21 +154,21 @@ func (t *Testnet) CreateTxClient(name,
 			Msg("error creating txsim")
 		return err
 	}
-
-	// copy over the keyring directory to the txsim instance
-	err = txsim.Instance.AddFolder(txsimKeyringDir, txsimRootDir, "10001:10001")
+	err = txsim.Instance.Commit()
 	if err != nil {
 		log.Err(err).
-			Str("directory", txsimKeyringDir).
 			Str("name", name).
-			Msg("error adding keyring dir to txsim")
+			Msg("error committing txsim")
 		return err
 	}
-	err = txsim.Instance.Commit()
+
+	// copy over the keyring directory to the txsim instance
+	err = txsim.Instance.AddFolder(txsimKeyringDir, txsimRootDir, "10001:10001")
```

Comment:
[microscopic nit]
it would be nice to define some constants for UID and GID and explain in their docs that these are the permissions given in the Celestia-app docker images